### PR TITLE
MINOR: Add Ruby Version File

### PR DESCRIPTION
**Pull request by original-brownbear**
Should make life a lot easier for people working on multiple Ruby projects + makes IDEA recognize the project's Ruby automatically :)

### CODE REVIEW

**Comment by original-brownbear** [comment]
seems to be incompatible with Jenkins :(

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+jruby-9.1.9.0
```
Comment (ph) [inline .ruby-version]: This should be jruby-9.1.10.0, our jenkins environment has that version installed.

Comment (original-brownbear) [inline .ruby-version]: @ph thanks, updated let's try :)

**Comment by original-brownbear** [comment]
@ph wdyt, good to merge (failure is the known millisecond resolution thing) ? :)

**Comment by original-brownbear** [comment]
@ph thanks :)

**Pull request closed without merging**